src/Entity/ReactionRuleConfig.php

```diff
@@ -121,11 +121,15 @@ class ReactionRuleConfig extends ConfigEntityBase {
   protected $module = 'rules';
 
   /**
-   * The event name this reaction rule is reacting on.
+   * The events this reaction rule is reacting on.
    *
-   * @var string
+   * Events array, key - numeric index, value - event array with next structure:
```

Comment:
Should be a full sentence alway:
The array is numerically indexed and contains arrays with the following structure: